add kvs example (#22)
ninech/deploio-examples: README.md

  ```

  See [dockerfile/java-kvs/README.md](dockerfile/java-kvs/README.md) for details and full setup instructions.

## KVS

> To make this example work, you need to first create a KVS instance:

Reviewer: ```suggestion
This example uses an [On-Demand Key-Value Store](https://docs.nine.ch/de/docs/on-demand-databases/on-demand-key-value-store/).To get started, you'll need to first create a KVS instance - either through `nctl` or via the [Cockpit](https://cockpit.nine.ch/en) interface: 
```

```bash
nctl create kvs <kvs-name>
```

> Once it is ready you can use the following command to create the app:

Reviewer: ```suggestion
Once your instance is ready you can create the app using the command below. You’ll need the instance’s connection details (FQDN and TOKEN), which can be retrieved using `nctl` or found in Cockpit under “Access Information.” For more details, see [Key-Value Store docs](https://docs.nine.ch/docs/on-demand-databases/on-demand-key-value-store#connecting):
```

```bash

> export KVSCLI_AUTH=$(nctl get kvs test --print-token)

Reviewer: ```suggestion
export KVS_PASSWORD=$(nctl get kvs test --print-token)
```
Maybe we could use `KVS_PASSWORD` here. This is what Marco did in hist java kvs example https://github.com/ninech/deploio-examples/blob/java-kvs/dockerfile/java-kvs/README.md. This is also how you name your Go variable `kvsPassword`.

export KVS_HOST=$(nctl get kvs test -o yaml | yq '.status.atProvider.fqdn')
nctl create application go \
  --git-url=https://github.com/ninech/deploio-examples \
  --git-sub-path=kvs \

>   --env="KVS_HOST=$KVS_HOST;KVSCLI_AUTH=$KVSCLI_AUTH"

Reviewer: ```suggestion
  --env="KVS_HOST=$KVS_HOST;KVS_PASSWORD=$KVS_PASSWORD"
```
